tests/test_permissions.py

```diff
@@ -25,6 +28,48 @@
 )
 
 
+@pytest.mark.django_db
+class TestCurrentOrgHelper(object):
```

Comment:
Aside from the `django_db` decorator, what is the advantage of having a test class here instead of independent test functions?

Reply:
@johnbaldwin because I like to group stuff into common namespaces so I save `test_current_org_helper_something_xyz`.

Reply:
Just to mention, a test module does not scope tests?

Reply:
true, I'll revisit my preferences. I could revisit it.